bylaws.html

```diff
@@ -404,11 +393,10 @@ <h3 id="types">3.8. Decision Types</h3>
     </tr>
     <tr>
       <td>New committer
-      <td>
-	Elect a new committer.
-	<br>
-	<br>
-	Nominations can be made by anyone by emailing <a href="https://mail-search.apache.org/pmc/private-arch/couchdb-private/">the private list</a>.
+      <td>Elect a new committer.
+        <br>
+        <br>
+        Anyone can make nominations by emailing <a href="https://mail-search.apache.org/pmc/private-arch/couchdb-private/">the private list</a>.
```

Comment:
We should probably take this opportunity to link to the new PonyMail `lists.apache.org` instead of the soon-to-be-retired old `mail-search.apache.org` link.

https://lists.apache.org/list.html?private@couchdb.apache.org (requires login)